Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -16,13 +17,34 @@
     /// </summary>
     public class RequestTrackingTelemetryModule : ITelemetryModule
     {
+        private const string RequestIdHeader = "AppInsights-RequestTrackingTelemetryModule-Request-Id";
```

Comment:
Please refrain from using `AppInsights` in favor to `ApplicationInsights` where possible